install/db/achecker_schema.sql

```diff
@@ -308,7 +308,7 @@ CREATE TABLE `themes` (
   `title` varchar(80) NOT NULL DEFAULT '',
   `version` varchar(10) NOT NULL DEFAULT '',
   `dir_name` varchar(20) NOT NULL DEFAULT '',
-  `last_updated` date NOT NULL DEFAULT '0000-00-00',
+  `last_updated` date DEFAULT NULL,
```

Comment:
There should always be a `last_updated` value. The original `NOT NULL` should be correct. Just remove `DEFAULT '0000-00-00'`.

Reply:
Same question as above

Reply:
`last_updated` here can be set to `DEFAULT NOT NULL` as its default value for the rows inserted in `achecker_schema.sql` does have the initial value. What do you think?

Reply:
I think it is ok here, would add a new commit now

Reply:
would remove the ```DEFAULT``` so that it would be just ```NOT NULL```, since it is not null, so no default value